content/altinn-studio/v10/getting-started/development/basic-form/_index.nb.md

```
@@ -1,39 +1,169 @@
 
-1. Klikk på __Opprett ny app__ øverst til høyre i skjermbildet.
-2. I feltet __Eier__ velger du hvem som skal eie appen. Hva du kan velge her, avhenger av om du har rett til å opprette en app for organisasjonen din, eller om appen blir registrert privat, på din bruker.
-   - Hvis du oppretter appen på din egen bruker, kan du teste all funksjonalitet i Altinn Studio, men du har ikke tilgang til 
-     testmiljøet.
-   - Hvis du har rettigheter til å opprette appen på en organisasjon, kan du også publisere tjenesten til
-     organisasjonens testmiljø.
-3. I feltet __Navn__ følger du reglene for appnavn og lager et kort og beskrivende navn på appen din.
-   _Navnet må ha minst to tegn og kan kun inneholde små bokstaver, tall og bindestrek. Navnet identifiserer appen din, og kan 
-   ikke endres etter at du har satt det. Du kan lage et eget visningsnavn etter at du har opprettet appen._
-4. Klikk på __Opprett app__.
+### Hva skal vi lage?
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)